infra/terraform/templates/launch.tf

```diff
@@ -55,9 +55,7 @@ resource "aws_autoscaling_group" "wellcomecollection_ecs_asg" {
   desired_capacity = 2
   min_size         = 2
   max_size         = 4
-  health_check_grace_period = 300
```

Comment:
I still need to make 100% sure - but no - there is no "ALB healthcheck" - I think if the instance is unhealthy - it get's scrapped and then the ALB brings one more into itself.

Let me 100% check.